src/components/status/Status.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { ParagraphSmall } from 'baseui/typography';
+import { useTheme } from '../../providers';
+import { borderRadius, padding } from '../../utils';
+
+export interface StatusProps {
+  value: string;
+  color?: string;
+  fontColor?: string;
+}
+
+export const Status = ({ value, color = '#E7E7E9', fontColor = '#0F1020' }: StatusProps) => {
```

Comment:
are there any constant values for these variables, something like "primary" or "secondary"? @hiddeottens maybe you know?

Reply:
Yea `#E7E7E9` can be imported from the `customColors` object as `light2` and `#0F1020` as `dark0`

Reply:
fixed